docs/examples/word_embedding/word_embedding_training.md

```diff
@@ -0,0 +1,292 @@
+# Word Embeddings Training and Evaluation
+
+```python
+import warnings
+warnings.filterwarnings('ignore')
+
+import itertools
+import time
+import math
+import logging
+import random
+
+import mxnet as mx
+import gluonnlp as nlp
+import numpy as np
+from scipy import stats
+
+# context = mx.cpu()  # Enable this to run on CPU
+context = mx.gpu(0)  # Enable this to run on GPU
+```
+
+### Data
+Here we use the Text8 corpus from the [Large Text Compression
+Benchmark](http://mattmahoney.net/dc/textdata.html) which includes the first
+100
+MB of cleaned text from the English Wikipedia.
+
+```python
+dataset = nlp.data.Text8()
+print('# sentences:', len(dataset))
+for sentence in dataset[:3]:
+    print('# tokens:', len(sentence), sentence[:5])
+```
+
+We pass the dataset to the `prepare_batches` function from the `utils.py` of
+this notebook.
+
+The following actions are taken:
+- subsampling
+- negative
+samples
+- subwords in CSR
```

Comment:
Many ppl are not familiar with mxnet csr. Add a link to https://mxnet.incubator.apache.org/tutorials/sparse/csr.html ?